frontend/src/scenes/data-management/revenue/CurrencyDropdown.tsx

```diff
@@ -1,38 +1,65 @@
-import { LemonSelect, LemonSelectOption, LemonSelectProps } from '@posthog/lemon-ui'
+import { LemonSelect, LemonSelectOption } from '@posthog/lemon-ui'
 import {
     CURRENCY_SYMBOL_TO_EMOJI_MAP,
     CURRENCY_SYMBOL_TO_NAME_MAP,
     DISABLED_CURRENCIES,
 } from 'lib/utils/geography/currency'
-import React from 'react'
 
 import { CurrencyCode } from '~/queries/schema/schema-general'
 
-type T = CurrencyCode | null
-type SelectProps = LemonSelectProps<T>
-type SelectOption = LemonSelectOption<T>
-type CurrencyDropdownProps = Omit<SelectProps, 'options'>
+type CurrencyDropdownProps = {
+    value: CurrencyCode | null
+    onChange: (currency: CurrencyCode | null) => void
+    visible?: boolean // Useful for stories to display the dropdown content
+}
+
+// These are the currencies that are most important to show first because they're used by the most customers.
+// Check our web analytics dashboard for the most popular countries from our visitors.
+const IMPORTANT_CURRENCIES: CurrencyCode[] = [
+    CurrencyCode.USD,
+    CurrencyCode.EUR,
+    CurrencyCode.GBP,
+    CurrencyCode.CAD,
+    CurrencyCode.INR,
+    CurrencyCode.CNY,
+    CurrencyCode.BRL,
+]
```

Comment:
Open to suggestions on whether we should add/remove something from here, I based myself off of our Web Analytics dashboard. In theory, CAD should rank above GBP, so consider this an homage to the King.

Reply:
Ha, actually, I could choose between USD/EUR and GBP/CAD depending on whether we're on our EU or US instance, but that's big scope creep, I won't go down that hole